### ISSUE TITLE
remove old weapon  v_model and w_models from the files.

### ISSUE THREAD

**Issue by condor00fr** [ValveSoftware/Source-1-Games#4279]
after the Love and war uptate, the old v_model/w_model system was discarded in favor of the current c_model format, however the model and texture files for thoses items and weapons are still in the game's files for some reason. so why not remove them to lighten the game's files a little bit?

**Comment by AzureAzel** [comment]
I believe the Invis Watches still use v_models/w_models. I don't know why. Possible oversight. Should consider porting those to c_models before any of those get wiped.

**Comment by condor00fr** [comment]
From what I know, the bottles, bats, watches and the disguise kit use theirs v_models because they where fully model unlike the rest. What I mean by that is that if you could inspect the other weapons they would show an incomplete model and lack textures on certain sides. Prehaps removing thoses as they don't have any use?

**Comment by Marioiscool246** [comment]
Removing these will also cause the placeholder ERROR model to show up in some community maps, primarily tr_walkway_rc2 since there's many props that use the older models.

**Comment by TF2CutContentWiki** [comment]
Yeah no, removing unused files from the game will cause many community maps to have error models and missing textures. Leave these files in the game.

**Comment by condor00fr** [comment]
Fair enough.
